SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceEvent.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+namespace SpatialGDK
+{
+struct FSpatialTraceEvent
+{
+	FSpatialTraceEvent(const char* InType, const FString& InMessage)
+		: Type(InType)
+		, Message(InMessage)
+	{
+	}
+
+	void AddData(FString Key, FString Value) { Data.Add(MakeTuple(MoveTemp(Key), MoveTemp(Value))); }
+
+	const char* Type;
```

Comment:
This seems very unsafe. I think this should be something that clearly owns the string instead of just a pointer to it.